src/rcc.rs

```diff
@@ -143,6 +151,33 @@ impl CFGR {
         self
     }
 
+    /// Enables write access to the registers in the backup domain
+    pub fn access_backup_domain(&mut self, pwr: &mut PWR) -> BackupDomain {
+        // NOTE: Exclusive access to RCC because this struct is a member of
+        // Rcc
+        let apb1_enr = unsafe { &(*RCC::ptr()).apb1enr };
```

Comment:
The Rcc struct is public, thus it can be destructured. Better to add this method directly on Rcc, and you then should have access without unsafe to apb1enr by the APB1 struct

Reply:
Good catch, it has been moved